src/main/resources/com/google/api/codegen/go/example.snip

```diff
@@ -22,13 +22,22 @@
     }
 
     @join method : view.apiMethods
-        @if method.isPageStreaming
-            {@pageStreamingMethod(view, method)}
-        @else
-            @if method.hasReturnValue
-                {@simpleMethod(view, method)}
+        @switch method.grpcStreamingType.toString
```

Comment:
you don't need the `.toString` part (which I didn't originally realize) (same in main.snip)